docs/core/install/how-to-detect-installed-versions.md

```diff
@@ -73,13 +73,17 @@ dotnet --list-sdks
 
 ## Check runtime versions
 
-You can see which versions of the .NET Core runtime are currently installed with the `dotnet --list-runtimes` command.
+You can see which versions of the .NET Core runtime are currently installed with the following command:
+
+```dotnetcli
+dotnet --list-runtimes
+```
+
+You get output similar to the following:
```

Comment:
Editors have indicated that `:` should only be used to to introduce a list, not for a snippet/code.

```suggestion
You get output similar to the following.
```